### ISSUE TITLE
Admins being able to cancel a financial contributions

### ISSUE THREAD

**Issue by shannondwray** [opencollective/opencollective#5946]
## Who is your user?
Admins 

## What are they trying to achieve?
Cancel a financial contribution 

There are lots of instances where it would be beneficial to be able to cancel a financial contribution for a particular user. We have a situation where a tier was deleted and it was assumed all financial contributions would be deleted in the process which didn't happen. It would be great if the admin could cancel the associated contributions. 

## How are they currently doing this?
We share instructions with the user on how to do this themselves. 

<details>
  <summary>Triage Template (core team only)</summary>

  This template is for members of the team to triage for prioritisation. For more guidance see https://www.loom.com/share/369ab467fbc64dec848085d38ff57ca0:

  P1 high frequency, high impact
  P2 low frequency, high impact
  P3 high frequency, low impact
  P4 low frequency, low impact

  Examples of high impact - a problem affects users during an essential process (expenses and payments > onboarding and registration > contributing) with no workaround.

  High frequency - >10% of users affected (measured as a proportion of total potential users for this case).
</details>

**Comment by shannondwray** [comment]
This would be great on Root Admin

**Comment by Betree** [comment]
This is already kind of possible by clicking on "Reject contribution" on the transactions page, but the feature is meant to prevent SPAM as it will also remove the membership.

Wouldn't be a big change in the API to support this.

**Comment by BenJam** [comment]
> already kind of possible by clicking on "Reject contribution"

++ does this also block the user at all? Might be too harsh otherwise.

**Comment by SudharakaP** [comment]
> > already kind of possible by clicking on "Reject contribution"
> 
> ++ does this also block the user at all? Might be too harsh otherwise.

Yes this does block the user, in the sense they cannot submit expenses anymore.

**Comment by znarf** [comment]
I don't see the link between rejecting contributions and submitting expenses.

You can inspect the behavior of rejectTransaction in:
https://github.com/opencollective/opencollective-api/blob/main/server/graphql/v2/mutation/TransactionMutations.ts#L92

**Comment by SudharakaP** [comment]
@znarf : Ops sorry, I somehow mistook the previous comment for rejecting an expense. But I realize now that this is about cancelling a contribution.

**Comment by znarf** [comment]
Rejecting an expense doesn't prevent the submitter to post new expenses either.

**Comment by mkurz** [comment]
Is this shipped? Can a  collective admin cancel a financial contribution?
